Makefile

```diff
@@ -80,8 +80,8 @@ clean: ## Clean locally generated Dockerfiles
 	-@rm -f $(COREDIR)/Dockerfile
 	-@rm -f $(WORKERDIR)/Dockerfile
 
-test: $(DOCKERCOMPOSE) client ## Run the client only and connect to an existing worker
-	$(DOCKERCOMPOSE) up $(UPARGS) client
+test: $(DOCKERCOMPOSE) client core ## Run the client only and connect to an existing worker
+	$(DOCKERCOMPOSE) up $(UPARGS) client core
```

Comment:
I plan to perform some cleanup of this Makefile after the merge, some of the existing functionality is bugging me, like lack of parallel builds with docker-compose.